### ISSUE TITLE
nwscript.nss seems to be stale

### ISSUE THREAD

**Issue by b5635** [urothis/nwn-assets#3]
https://github.com/urothis/nwn-assets/blob/main/nss/nwscript.nss

Ctrl + F "GetDomain" the function isn't defined, which was added last September I think in EE. My repo used to pull this repo to use the scripts to compile for nasher, but it seems like compilation is failing due to an outdated nwscript.nss

I was using this version of the repo previously, which has the correct version: https://github.com/urothis/nwn-assets/tree/ae5d91a4a5dde5cba66deb4e305cf846a45af9cc

**Comment by urothis** [comment]
@b5635 I just pushed a fix to update the nwscript.nss. 

I'm digging into why it's pulling an older version, I swapped to a new local workflow for this repo so I could include models and other assets not included in the beamdog docker image. But I might need to have the docker nwscript overwrite the local after I obtain the other files.

I'll take a deeper look at this today, but nwscript.nss should be up to date.
